web/src/utils/atlas/fetchUser.ts

```diff
@@ -0,0 +1,50 @@
+import { gql, type GraphQLClient } from "graphql-request";
+import { toast } from "react-toastify";
+
+import { OPTIONS } from "utils/wrapWithToast";
+
+type GetUserResponse = {
+  user: {
+    email: string;
+  };
+};
+
+export type User = {
+  email: string;
+};
+
+const query = gql`
+  query GetUser($address: String!) {
+    user(address: $address) {
+      email
+    }
+  }
+`;
+
+export function fetchUser(client: GraphQLClient, address: string): Promise<User> {
+  const variables = {
+    address,
+  };
```

Comment:
This is no longer required. Even more, will rise an error because address is not an allowed argument